tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "lib": [
+      "esnext"
+    ],
+    "outDir": "./lib",
+    "target": "esnext",
+    "declaration": true,
+    "allowSyntheticDefaultImports": true,
+    "module": "none"
```

Comment:
Try to use here `CommonJS` value instead of `none`. It's responsible for the output format.